WordPress/gutenberg
# [RNmobile] Trim feature image copy

## Description
This is a partial fix for https://github.com/wordpress-mobile/gutenberg-mobile/issues/4432
Leaving whitespace at the end of a string causes the translation look up to fail on Android.
While this is not the most robust solution it does fix the two known problem strings.

## How has this been tested?



Since this issue only occurs on Android, testing should be done with an Android device or emulator
1. Change the system language to one other than English.
2. Open the WordPress app and open or create a post with an image block
3. Open the image block settings
4. Notice that default copy "Set as Featured Image" is translated to the system language 
5. Set the image as the Featured image
6. Re open the image settings
7. Notice that the default copy "Remove as Featured Image" is translated to the system language

## Types of changes




- Remove trailing whitespace from two localized strings

## Checklist:
